### PR TITLE
Removing out of date examples

### PR DESCRIPTION
**Pull request by billyjacobson**
Removing out of date Node and Dotnet examples
Updating README.md with links to new sample code

There is more to do, but I'm just trying to do a bit at a time

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+---
+![status: inactive](https://img.shields.io/badge/status-inactive-red.svg)
+
+###Note: Examples have moved
+
+
+Google Cloud Bigtable examples samples have moved. This directory is no
```
Comment (kolea2) [inline README.md:7]: I'd remove this for now

Comment (igorbernstein2) [reply README.md:7]: +1

File: README.md [hunk 2]
```diff
@@ -62,6 +60,20 @@ infrequent product and client library announcements.
 See [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to contribute
 to this project.
 
+## More examples
+
+More Google Cloud Bigtable examples are available in in the following languages:
```
Comment (jaceklaskowski) [inline README.md:65]: Looks like `in`s doubled.

Comment (billyjacobson) [reply README.md:65]: Fixed by #342

**Pull request merged**